src/services/admin/admin.tsx

```
@@ -0,0 +1,101 @@
+
+export interface Mapping {
+	name?: string;
+	label?: string;
+	documentSubType?: string;
+	docType?: string;
+	fieldName?: string;
+	documentMappings?: DocumentFieldMapping[];
+	fieldValueNormalizationMapping?: FieldValueNormalization;
+}
+
+interface MappingRequest {
```

ESLint

[error] 'MappingRequest' is defined but never used.

(@typescript-eslint/no-unused-vars)